spec/models/live_metric_spec.rb

```diff
@@ -32,7 +32,7 @@
   end
 
   it "#process_conditions" do
-    processed = LiveMetric.process_conditions(conditions)
+    processed = LiveMetric.process_conditions([conditions])
```

Comment:
Adressed.
parse_conditions works over raw_conditions and process_conditions on the wrapped one.
This is updated on the test.